modules/contracts/adapters/dex/src/handlers/execute.rs

```diff
@@ -54,7 +54,7 @@ pub fn execute_handler(
             // unwrap namespace, since it's unlikely to have unclaimed abstract namespace
             let namespace_info = namespace.unwrap();
             ensure_eq!(
-                namespace_info.account_base.proxy,
+                namespace_info.account_base.manager,
```

Comment:
This would be weird. The manager can only call it if it has the module installed.

Can we change this to use the accounts's admin as the accepted sender? Like we do for apps?

Reply:
Sure, fixed!